articles/active-directory/develop/scenario-daemon-app-configuration.md

```diff
@@ -136,6 +136,9 @@ IConfidentialClientApplication app;
 
 ```python
 import msal
+import json
+import sys
+import logging
```

Comment:
All the changes in this PR are harmless, even this unnecessary line of `import logging` (which is never used in this doc). Thus I'm still voting for accepting it.

But based on the changes to these import statements, I also see a trend of customers attempting copy & paste from this doc and expecting it would work. This brings maintenance burden which is not scalable.

Our doc should mention something like this at the beginning or at the end: "A fully tested and runnable sample is available here: https://github.com/Azure-Samples/ms-identity-python-daemon ".